runtime/oti/bcverify_api.h

```diff
@@ -206,6 +206,58 @@ j9bcv_satisfyClassLoadingConstraint (J9VMThread* vmThread, J9ClassLoader* loader
 void 
 unlinkClassLoadingConstraints (J9JavaVM* jvm);
 
+/* ---------------- classrelationships.c ---------------- */
+
+/**
+ * @brief						Record a class relationship in the class relationships table.
+ *
+ * @param *vmThread				The calling vmThread
+ * @param *classLoader			Class loader to record the relationship to
+ * @param *childName			Class name of the child (source class) to record
+ * @param childNameLength		Length of the child class name
+ * @param *parentName			Class name of the parent (target class, i.e. superclass or interface) to record
+ * @param parentNameLength		Length of the parent class name
+ * @param *reasonCode			Set to BCV_ERR_INSUFFICIENT_MEMORY if a child entry or parent node cannot be allocated, otherwise 0
+ * @return IDATA				Returns TRUE if successful and FALSE if an out of memory error occurs
+ */
+IDATA
+j9bcv_recordClassRelationship(J9VMThread *vmThread, J9ClassLoader *classLoader, U_8 *childName, UDATA childNameLength, U_8 *parentName, UDATA parentNameLength, IDATA *reasonCode);
+
+/**
+ * @brief						Validate each recorded relationship for a class (child).
+ * 								Parent classes are expected to be interfaces, superclasses or superinterfaces of the child class.
+ *
+ * @param *vmThread				The calling vmThread
+ * @param *classLoader			Class loader to look up relationships from
+ * @param *childName			Class name of the child class to validate
+ * @param childNameLength		Length of the child class name
+ * @param childClass			The loaded child J9Class
+ * @return J9Class				Returns NULL if successful, or the class that fails validation if unsuccessful
+ */
+J9Class *
+j9bcv_validateClassRelationships(J9VMThread *vmThread, J9ClassLoader *classLoader, U_8 *childName, UDATA childNameLength, J9Class *childClass);
+
+/**
+ * @brief						Allocates new hash table to store class relationship entries.
+ *
+ * @param *classLoader			The class loader where the hash table is stored
+ * @param *vm					The VM requesting a new class relationship hash table
+ * @return J9HashTable			Returns 0 if successful, and 1 otherwise.
+ */
+UDATA
+j9bcv_hashClassRelationshipTableNew(J9ClassLoader *classLoader, J9JavaVM *vm);
+
+/**
+ * @brief						Frees memory for each J9ClassRelationship table entry, J9ClassRelationshipNode, 
+ * 								and the classRelationships hash table itself.
+ *
+ * @param *vmThread				The calling VM thread
+ * @param *classLoader			The class loader where the hash table is stored
+ * @param *vm					The VM requesting deallocation of the class relationship hash table
+ * @return void
```

Comment:
`* @return void` can be deleted.

Can you reformat the comments so there isn't so much white space between the parameter and it's description?  A single space (or tab) would be sufficient.